compiler/qsc_parse/src/prim.rs

```diff
@@ -77,6 +77,8 @@ pub(super) fn ident(s: &mut ParserContext) -> Result<Box<Ident>> {
     }
 }
 
+/// This function takes a [Path] and converts it into a single ident, which
```

Comment:
nitpick: I don't see this as "taking" a path since the path is not an argument, but rather parsing the path from the given stream context.